test/src/comps/test_qb.cc

```diff
@@ -340,14 +337,12 @@ template <typename T>
         gemm<T>(Layout::ColMajor, Op::NoTrans, Op::NoTrans, m, n, k_est, -1.0, Q_dat, m, B_dat, k_est, 1.0, A_dat, m);
         
         T norm_test_1 = lange(Norm::Fro, m, n, A_dat, m);
-        if(tol == 0.0)
-        {
+        if(tol == 0.0) {
             // Test Zero Tol Output
             printf("FRO NORM OF A - QB:    %e\n", norm_test_1);
             ASSERT_NEAR(norm_test_1, 0, 1e-12);
```

Comment:
Tolerance should be relative to the current numeric type.

Reply:
Fixed.